azurerm/internal/services/appplatform/spring_cloud_service_resource.go

```diff
@@ -110,6 +111,22 @@ func resourceArmSpringCloudService() *schema.Resource {
 							Computed: true,
 							ForceNew: true,
 						},
+
+						"outbound_ip": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"public_ips": {
+										Type:     schema.TypeList,
+										Computed: true,
+										Elem: &schema.Schema{
+											Type: schema.TypeString,
+										},
+									},
```

Comment:
there's no benefit in this value being nested, so can we move this to a top level field?

Reply:
I will do this after we made a conclusion of the above issue

Reply:
have moved to a top level field